lib/node_modules/@stdlib/array/base/count-ifs/benchmark/benchmark.length.js

```diff
@@ -0,0 +1,123 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var isNonNegativeInteger = require( '@stdlib/assert/is-nonnegative-integer' ).isPrimitive;
+var bernoulli = require( '@stdlib/random/array/bernoulli' );
+var pkg = require( './../package.json' ).name;
+var countIfs = require( './../lib' );
+
+
+// FUNCTIONS //
+
+/**
+* First predicate function.
+*
+* @private
+* @param {number} v - value
+* @returns {boolean} result
+*/
+function predicate0( v ) {
+	return v > 0.0;
+}
+
+/**
+* Second predicate function.
+*
+* @private
+* @param {number} v - value
+* @returns {boolean} result
+*/
+function predicate1( v ) {
+	return v < 0.0;
+}
+
+/**
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} len - array length
+* @returns {Function} benchmark function
+*/
+function createBenchmark( len ) {
+	var x0 = bernoulli( len, 0.5, {
```

Comment:
Based on your predicates, I think you want `uniform( len, -1.0, 1.0, {...} )` here. Bernoulli can only be `0` or `1`, but `predicate1` is checking for less than `0`.